core/src/test/java/org/opensearch/sql/expression/datetime/PeriodFunctionsTest.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.sql.expression.datetime;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
+
+import java.util.stream.Stream;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.opensearch.sql.expression.DSL;
+
+public class PeriodFunctionsTest extends DateTimeTestBase {
+
+  /**
+   * Generate sample data for `PERIOD_ADD` function.
+   * @return A data set.
+   */
+  public static Stream<Arguments> getTestDataForPeriodAdd() {
+    return Stream.of(
+        Arguments.of(1, 3, 200004), // Jan 2000 + 3
```

Comment:
Is this April 2000 instead of Jan 2000?

Reply:
Last argument is the expected result value.